doc/install_ubuntu.txt

```diff
@@ -12,6 +12,31 @@ For NVIDIA Jetson TX1 embedded platform:
     sudo apt-get install python-numpy python-scipy python-dev python-pip python-nose g++ libblas-dev git
     pip install --upgrade --no-deps git+git://github.com/Theano/Theano.git --user  # Need Theano 0.8(not yet released) or more recent
 
+For Ubuntu 16.04 with cuda 7.5
+
+.. code-block:: bash
+
+    sudo apt-get install python-numpy python-scipy python-dev python-pip python-nose g++ libopenblas-dev git
+    sudo pip install Theano
```

Comment:
If you want to make a PR to add in the doc how to use virtualenv, we can add that. But don't forget that many people use anaconda that is free and link with mkl. That is even faster.